diff -Naur wafp-0.01-26c3/lib/wafp_https.rb wafp-0.01-26c3-fhs//lib/wafp_https.rb
--- wafp-0.01-26c3/lib/wafp_https.rb	2009-12-29 01:01:27.000000000 +0100
+++ wafp-0.01-26c3-fhs//lib/wafp_https.rb	2011-05-19 11:54:02.000000000 +0200
@@ -98,7 +98,7 @@
 
 =end
 
-require "lib/wafp_http.rb"
+require "/usr/share/wafp/lib/wafp_http.rb"
 require 'openssl'
 
 module Net
diff -Naur wafp-0.01-26c3/wafp.rb wafp-0.01-26c3-fhs//wafp.rb
--- wafp-0.01-26c3/wafp.rb	2009-12-29 01:01:27.000000000 +0100
+++ wafp-0.01-26c3-fhs//wafp.rb	2011-05-19 11:53:17.000000000 +0200
@@ -45,10 +45,10 @@
 require 'getoptlong'
 require 'sqlite3'
 # this is a modified version of net/http
-require "lib/wafp_http.rb"
-require "lib/wafp_https.rb"
+require "/usr/share/wafp/lib/wafp_http.rb"
+require "/usr/share/wafp/lib/wafp_https.rb"
 # this is a modified version of pidify
-require "lib/wafp_pidify.rb"
+require "/usr/share/wafp/lib/wafp_pidify.rb"
 require 'uri'
 require 'thread'
 require 'timeout'
@@ -61,8 +61,8 @@
 CONTACT      = 'richard.sammet@gmail.com';
 WEBSITE      = 'http://mytty.org/wafp/';
 # the databases
-FDB          = 'fprints_wafp.db'
-SDB          = 'scan_wafp.db'
+FDB          = '/usr/share/wafp/fprints_wafp.db'
+SDB          = '/usr/share/wafp/scan_wafp.db'
 # predefined http headers
 USERAGENT    = 'Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.9.0.10) Gecko/2009050223 Gentoo Firefox/3.0.10'
 
